TranscriptAnnotator.pm

```diff
@@ -0,0 +1,260 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ TranscriptAnnotator.pm
+
+=head1 SYNOPSIS
+
+ mv TranscriptAnnotator.pm ~/.vep/Plugins
+ ./vep -i variations.vcf --plugin TranscriptAnnotator,file=/path/to/file.txt.gz
+
+=head1 DESCRIPTION
+
+ A VEP plugin that annotates variant-transcript pairs based on a given file:
+
+   --plugin TranscriptAnnotator,file=${HOME}/file.tsv.gz
+
+ Example of a valid tab-separated annotation file (before bgzip and tabix):
```

Comment:
You should also add instructions on how to prepare the file to be used by the plugin.

Reply:
Added the following instructions, what do you think?

```
 Please bgzip and tabix the file with commands such as:

   bgzip file.txt
   tabix -b2 -e2 file.txt.gz
```

Reply:
Yes, something like that.